client/api_account.go

```diff
@@ -21,6 +22,7 @@ type Account interface {
 
 	CreatePaymentAccount(ctx context.Context, address string, txOption *gnfdSdkTypes.TxOption) (string, error)
 	Transfer(ctx context.Context, toAddress string, amount math.Int, txOption gnfdSdkTypes.TxOption) (string, error)
+	MultiTransfer(ctx context.Context, details []types.TransferDetail, txOption *gnfdSdkTypes.TxOption) (string, error)
```

Comment:
The txOption will be modified when calling this interface?

If not, I suggest using value, not pointer, to make the interface style unified.